iceoryx_posh/test/integrationtests/test_service_discovery.cpp

```diff
@@ -40,10 +40,37 @@ using iox::runtime::ServiceContainer;
 class ServiceDiscovery_test : public RouDi_GTest
 {
   public:
+    void SetUp() override
+    {
+        searchResultOfFindServiceWithLambda.clear();
+    }
+
+    void TearDown() override
+    {
+    }
+
     iox::runtime::PoshRuntime* runtime{&iox::runtime::PoshRuntime::initRuntime("Runtime")};
     ServiceDiscovery sut;
+
+    static ServiceContainer searchResultOfFindServiceWithLambda;
+    static void lambda(const ServiceContainer& s)
```

Comment:
Nitpick: This is not a lambda but a static function.
```suggestion
    static void staticFunction(const ServiceContainer& s)
```

Reply:
Right, I would usually avoid calling these constructs like this in the code anyway and prefer handler/function/descriptive name (as it is a theoretical concept from https://en.wikipedia.org/wiki/Lambda_calculus).